in_toto/gpg/common.py

```diff
@@ -660,6 +718,7 @@ def parse_signature_packet(data, supported_signature_types=None,
   # conflict resolution scheme that makes more sense.
   # (see RFC4880 5.2.4.1.)
   # Below we only consider the last and favor hashed over unhashed subpackets
+  # TODO: Should we warn if a we use an unhashed subpacket?
```

Comment:
I think this would be a nice-to-have if possible.

Reply:
Yes, it would be. But I'm not sure where to best place the warning. Would it be okay to just keep the `TODO` note for now and revisit at a later time?